packages/web/src/utils/clipboardUtil.ts

```diff
@@ -30,7 +30,11 @@ export const copyToClipboard = (str: string) => {
     el.select()
   }
 
-  document.execCommand('copy')
+  if (navigator.clipboard) {
```

Comment:
what's this

Reply:
this is the new way to copy to clipboard. document.execCommand('copy') is deprecated. Wanted to keep it backwards compatible though